### PR TITLE
Add coturn server

### PR DESCRIPTION
**Pull request by Sheeo**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -16,6 +16,17 @@ services:
...
   #
+  # Coturn server for proxying between players
+  # It uses net: "host" for performance reasons.
+  #
+  coturn:
+    image: bprodoehl/turnserver
+    network_mode: "host"
+    environment:
+    - SKIP_AUTO_IP=true
+    - EXTERNAL_IP=${EXTERNAL_IP}
```
Comment (micheljung) [inline docker-compose.yml]: * The service should be called `faf-coturn` to keep up with the naming convention
* The environment variables should be defined in `config.template/faf-coturn/faf-coturn.env`, where `EXTERNAL_IP` should be set to something generic like `127.0.0.1`
* `EXTERNAL_IP` can then be removed from `.env`

**Comment by Sheeo** [comment]
Updated as well as updated the host-network to be docker-swarm compatible.

**Comment by micheljung** [comment]
Looks like travis' docker compose doesn't support 3.3 yet. Can we stick with 3.2? Otherwise we need to tell travis to update it

**Comment by Sheeo** [comment]
That precludes us from using the configs feature--though I haven't started with that yet, I'd rather like to use it. We can manually update docker until they push an update to their env.

**Comment by Sheeo** [comment]
Seemingly it was mainly `docker-compose` being out of date; the docker engine is rather old however. It's good to install the latest so we're testing with exactly what we'll be deploying on

**Pull request merged**